build.gradle

```diff
@@ -45,8 +46,8 @@ subprojects {
     apply plugin: 'java-library'
     apply plugin: 'org.inferred.processors'
 
-    sourceCompatibility = 1.8
-    targetCompatibility = 1.8
+    sourceCompatibility = 11
+    targetCompatibility = 11
```

Comment:
this probably warrants a callout in the changelog, that palantir-java-format is now only able to run on java11+

Reply:
For sure! Might even publish a new major release so 2.x is for jdk11+